packages/opc-back-to-top/demo/index.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>opc-back-to-top</title>
+    <script src="opc-back-to-top.js"></script>
+    <style> 
+    div{
+        height: 100vh;
+        background-color: #C0C0C0;
+    }
+     </style>
+</head>
+<body>
+    <div></div>
+        <opc-back-to-top></opc-back-to-top>
+        <opc-back-to-top>Back to Top</opc-back-to-top>
+        <opc-back-to-top reveal-at="200"></opc-back-to-top>
+        <opc-back-to-top reveal-at="300"></opc-back-to-top>
```

Comment:
Can please add an example with a slot. and updated the screenshot & readme according to it.

Reply:
There is an example with slot also <opc-back-to-top>Back to Top</opc-back-to-top>  can you please elaborate what do you mean by slot ?

Reply:
an example with slot.
```suggestion
        <opc-back-to-top reveal-at="300">
             <div>go to top</div>
        </opc-back-to-top>
```

Reply:
@gisumit  can you plz check.

Reply:
fixed by anjanee